domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/API.IntegrationTests.csproj

```diff
@@ -10,6 +10,8 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="Audit.EntityFramework.Core" Version="21.0.1" />
+        <PackageReference Include="Audit.NET" Version="21.0.1" />
```

Comment:
I do not think it is needed to add these package references

Reply:
You are correct. removed